hoodie-client/src/test/java/com/uber/hoodie/index/bloom/TestHoodieGlobalBloomIndex.java

```diff
@@ -0,0 +1,560 @@
+/*
```

Comment:
Not sure if we should repeat all tests here, let me think about it a little more.

Reply:
+1 @leletan I think just one test case with partition paths being search not same as input might be good enough here..

Reply:
@n3nash what do you think?

Reply:
Yes, I think that should be enough.